meeting-notes/2018/2018-08-20--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,204 @@
+ - Done:
+   - IPLD:
+     - thinking through go-ipld interfaces from a performance perspective (no concrete progress)
+     - have a plan to fix the interfaces and move to the IPLD org: https://github.com/ipld/go-ipld/issues/2
+   - fix (in review): stop go-libp2p from NAT port mapping non-nat (e.g., localhost) addresses.
+   - fix (in review): stop go-ipfs from listening on random ports.
+   - misc fixes, reviews, gx
+ - Next:
+   - multistream-select (again)
+   - continue work on IPLD interfaces/migration.
+ - Blocked:
+   - @daviddias: [Remove bitswap unwant](https://github.com/ipfs/go-ipfs/pull/5308)
+   - @daviddias: [New multiaddr protocol](https://github.com/multiformats/multiaddr/pull/68)
+ - Ask:
+   - Please review PRs!
+   - Bug me if you're blocked on something.
+
+@Magik6k
+ - Done:
+   - Coverage from Jenkins - https://github.com/ipfs/go-ipfs/pull/5389
+     - Needs a review + some admin actions
+   - Rebasing / finishing PRs
+ - Blocked:
+ - Next:
+   - get go-cidutil to be released on gx for coreapi/dht - https://github.com/ipfs/go-ipfs/pull/4804
+     - + update the PR to use it
+   - More coreapi command refactoring
+   - `ipfs name resolve --stream` (@stebalien says ❤️)
+   - workaround peerHost.SetStreamHandler not returning errors by using SetStreamHandlerMatch instead (for `ipfs p2p` refactor)
+
+@keks
+ - Done:
+   - cmds refactor is on it's way
+     - development branch of go-ipfs now uses newest version of refactored cmds lib
+     - sharness tests fail
+   - start of discussion on gx-gomod
+     - https://github.com/whyrusleeping/gx/issues/200
+ - Blocking:
+   - keep gx-gxmod discussion alive (not urgent on my side)
+   - get https://github.com/ipfs/go-ipfs/pull/5035 reviewed (not *super* urgent)
+ - Next:
+   - fix sharness tests
+   - continue designing gx-gomod
+
+@hsanjuan
+ - Done:
+   - Another round. Refs -r: https://github.com/ipfs/go-ipfs/pull/5337
+   - Extract MFS (waiting OK): https://github.com/ipfs/go-ipfs/pull/5391
+ 
+ - Blocked:
+ - Next:
+ 
+@bigs
+ - Done:
+   - Reversible netdefs and subnet support (feat/subnets on go-netdef)
+   - Update existing netdef tool
+   - Coral pairing with @carospiegly
+ 	 - Merged go-datastore and go-ds-badger changes!
+ - Blocked:
+ - Next:
+ 	 - Link options for subnet-to-subnet links in netdef
+   - Collision-free namespacing for netdef
+   - Helping boot up @raulk on Peerstore chnages
+ - Notes:
+   - OOO second half Thursday, all Friday
+
+@mgoelzer
+ - Done:
+   - OKRs - wrangling everyone for mid-quarter scores; mid-qtr report to LabOS
+   - Scheduling some recruiting meetings for September
+   - Recruiting spreadsheet for Thr meeting
+
+
+ - Blocked:
+ 	- Not blocked -- just not enough hours in the day!
+
+
+ - Next:
+   - Plan September trip to Bay Area to maximize chances to recruiting 1 or more great Go engineer s
+   - Waffle cleanup
+   - Continue developing go/js/rust interop demo
+   - Docs improvement SOW and interviews
+   - Rust dev shop SOWs
```

Comment:
@mgoelzer I don't have any particular thing against sharing this kind of update here, but have in mind that this is a call focused for Dev of the Go implementations of IPFS, libp2p and IPLD and not for PM'ing or Protocol Labs

Reply:
👍 - noted for future ref